### PR TITLE
Enable DUAL_STACK flags at test framework level

### PR DESCRIPTION
**Pull request by kfaseela**
Currently the tests are run without enabling this flag for majority of tests. Only some of the pre-submit
tests were enabling the flag. So, let us try to enable the required DUAL_STACK flags at the framework level.

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
LGTM

**Comment by zhlsunshine** [comment]
@kfaseela, LGTM,  and I think the next thing is to create the dual stack service to do the integration test. Good work!

File: tests/integration/security/main_test.go [hunk 1]
```diff
@@ -60,21 +59,6 @@ meshConfig:
     gatewayTopology:
       numTrustedProxies: 1 # Needed for X-Forwarded-For (See https://istio.io/latest/docs/ops/configuration/traffic-management/network-topologies/)
 `
-			} else {
-				cfg.ControlPlaneValues = `
-values:
-  pilot: 
-    env: 
-      PILOT_JWT_ENABLE_REMOTE_JWKS: true
```
Comment (howardjohn) [inline tests/integration/security/main_test.go:68]: I think we are losing this and L74 that we still want

Comment (howardjohn) [reply tests/integration/security/main_test.go:68]: interesting the tests pass...

Comment (kfaseela) [reply tests/integration/security/main_test.go:68]: It was already there in the if part too.. I just removed the if else..

**Pull request merged**